### PR TITLE
eclass-writing: Add default src_configure to jmake example

### PR DESCRIPTION
**Pull request by laumann**
Signed-off-by: Thomas Bracht Laumann Jespersen <t@laumann.xyz>

### CODE REVIEW

File: eclass-writing/text.xml [hunk 1]
```diff
@@ -667,7 +667,7 @@ src_compile() {
 <body>
 
 <p>
-A simple eclass which defines a number of functions and a default
+A simple eclass which defines a number of functions and default <c>src_confgure</c> and
 <c>src_compile</c> for the (hypothetical) <c>jmake</c> build system might look
 something like the following:
```
Comment (laumann) [inline eclass-writing/text.xml:672]: I think this paragraph could be better, I could rework it a bit.

File: eclass-writing/text.xml [hunk 2]
```diff
@@ -686,7 +686,7 @@ something like the following:
 # Demonstrates EXPORT_FUNCTIONS and defines simple wrappers for the
 # (hypothetical) jmake build system and a default src_compile.
```
Comment (thesamesam) [inline eclass-writing/text.xml:687]: We should update this comment too.

File: eclass-writing/text.xml (hunk 1 above)
Comment (thesamesam) [reply eclass-writing/text.xml:672]: What are you thinking?

File: eclass-writing/text.xml (hunk 2 above)
Comment (laumann) [reply eclass-writing/text.xml]: Done.

**Comment by laumann** [comment]
@thesamesam Rebased on master as well.

File: eclass-writing/text.xml (hunk 1 above)
Comment (thesamesam) [reply eclass-writing/text.xml:672]: Yeah, let's throw that in. Separate commit.

It reads easier :)

Comment (laumann) [reply eclass-writing/text.xml]: Done.

**Review by thesamesam — APPROVED** [review]
Much better, thank you!

**Pull request closed without merging**